scripts/check_frontend_coverage.py

```diff
@@ -290,11 +290,12 @@
     'Polyfills.ts',
     'post-publish-modal.controller.ts',
     'practice-tab.directive.ts',
+    'preferences-page.component.ts',
```

Comment:
We don't expect to add to this list. If it wasn't here earlier, please add tests to completely cover this file also.